docs/UserGuide.md

```diff
@@ -755,7 +760,23 @@ Furthermore, certain edits can cause the AddressBook to behave in unexpected way
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous MediContact home folder.
+
+**Q**: What should I do if I input an invalid command?<br>
+**A**: Check out [Features](#features) or simple run `help` to see the full list of commands, and try entering a valid command again.
+
+**Q**: Where should I start if I am a novice to CLI?<br>
+**A**: Run `help` to check the format. For more information on each command's usage, check out the [Features](#features) section.
+
+**Q**: Can I have duplicate contacts in my address book? <br>
+**A**: Duplicate names are not permitted. Contacts with the same name, even if capitalized differently, are considered duplicates.
+
+**Q**: Can I mark certain patients as high-priority?<br>
```

Comment:
Can consider rewording this question to marking patients as 'favourite' or 'more important` instead. High-priority could potentially imply that there's a feature allowing them to set patient contacts as low, medium or high priority.